src/composables/pools/usePoolCreation.spec.ts

```diff
@@ -6,7 +6,14 @@ import usePoolCreation, { PoolSeedToken } from './usePoolCreation';
 
 const tokens: Record<string, PoolSeedToken> = {};
 
-jest.mock('vue-i18n');
+jest.mock('vue-i18n', () => ({
```

Comment:
You could still keep the mock file and stop it from auto-mocking in other files with `jest.unmock('vue-i18n');` in the main jest file, as node_modules are always mocked by default.

Reply:
👍 Fixed